### ISSUE TITLE
Finally a real world use for Eagles hidden API

### ISSUE THREAD

**Issue by jasondavis** [turnercore/s3agle#1]
I’ve been using Eagle for at least 7 years or from its infancy stage and It is a real Gem that everybody should know about.  Just as Obsidian is, I put Eagle up in the same list of amazingly useful software.

I just want to publicly say thank you for connecting 2 of my favorite apps.
 
 As a Windows user I also religiously use ShareX and have been wanting to utilize Eagles API to have my screenshots and media uploads from my ShareX app also be added to my Eagle library using the Eagle API.  I haven’t had time to do it yet though and was hoping to see a real life use of the Eagle API first and this plugin checks that box for me as well.

So Thank you for sharing 🙏 very cool

**Comment by turnercore** [comment]
@jasondavis , Thanks so much, this made my day. I've been very busy with life lately, but I should have some updates that will make it run better. It ran on the dev instances and when I tried to download it from the plugin store I started getting an error. If it doesn't work for you then check back in in about a week and it should. Let me know if you find any issues with the plugin, or have any questions on the Eagle API, that part seemed to work well.
